packages/components/input/__tests__/input.test.tsx

```diff
@@ -146,7 +146,11 @@ describe('Input.vue', () => {
   })
 
   test('rows', () => {
-    const wrapper = mount(() => <Input type="textarea" rows={3} />)
+    const wrapper = mount(() => {
+      // type checking failed as `rows` is a fallthrough attribute
+      // @ts-ignore
```

Comment:
This annotation should be able to be removed now

Reply:
Removed